addons/website_forum/static/src/js/website_forum.js

```diff
@@ -408,10 +398,20 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
     _onDeleteCommentClick: function (ev) {
         ev.preventDefault();
         var $link = $(ev.currentTarget);
+        var $container = $link.closest('.o_wforum_post_comments_container');
+
         this._rpc({
             route: $link.closest('form').attr('action'),
         }).then(function () {
-            $link.parents('.comment').first().remove();
+            $link.parents('.o_wforum_post_comment').first().remove();
+
+            if ($container.find('.o_wforum_post_comment').length > 0) {
+                $container.find('.o_wforum_comments_count').text(function () {
+                    return parseInt($(this).text()) - 1;
```

Comment:
Any reason it is not the value `$container.find('.o_wforum_post_comment').length` which is used here ?

Reply:
Sorry, do you mean "in which case the condition is not true?"
If you remove a comment and it's the last one

Reply:
Yes, in this case is ok, you can replace it with `$container.find('.o_wforum_post_comment').length`